lab-taylor/__test__/gallery.test.js

```diff
@@ -0,0 +1,365 @@
+'use strict';
+
+const request = require('superagent');
+const serverToggle = require('../lib/server-toggle.js');
+const server = require('../server.js');
+
+const User = require('../model/user.js');
+const Gallery = require('../model/gallery.js');
+
+require('jest');
+
+const url = 'http://localhost:3000';
+
+const exampleUser = {
+  username: 'Teacup Timmy',
+  password: '1234',
+  email: 'teacuptimmy@test.com',
+};
+
+const exampleGallery = {
+  name: 'exampleGallery',
+  desc: 'some description',
+};
+
+const galleryUpdate = {
+  name: 'update gallery',
+  desc: 'update description',
+};
+
+describe('Gallery Routes', function() {
+  beforeAll( done => {
+    serverToggle.serverOn(server, done);
+  });
+
+  afterAll( done => {
+    serverToggle.serverOff(server, done);
+  });
+
+  afterEach( done => {
+    Promise.all([
+      User.remove({}),
+      Gallery.remove({}),
+    ])
+      .then( () => done())
+      .catch(done);
+  });
+
+  describe('POST: /api/gallery', () => {
+    describe('with a valid body', () => {
+      beforeEach( done => {
+        new User(exampleUser)
+          .generatePasswordHash(exampleUser.password)
+          .then( user => user.save())
```

Comment:
I would take out these spaces before your first args. Why a space on the left but not on the right? A tad inconsistent.